package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "name": "masterPortfolio",
+  "name": "master-portfolio",
```

Comment:
Can you revert this, please? The name should be `masterPortfolio` IMO. It's too long since we started using that name and it would spoil the SEO of website.

Reply:
What do you mean 'revert this', revet the commit?